release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -569,6 +569,11 @@ A Node Tuning Operator (NTO) metric is now added to Telemetry. Follow the proced
 
 The Node Feature Discovery (NFD) Topology Updater is a daemon responsible for examining allocated resources on a worker node. It accounts for resources that are available to be allocated to new pod on a per-zone basis, where a zone can be a Non-Uniform Memory Access (NUMA) node. See xref:../hardware_enablement/psap-node-feature-discovery-operator.adoc#using-the-nfd-topology-updater_node-feature-discovery-operator[Using the NFD Topology Updater] for more information.
 
+
+[id="ocp-4-10-hyper-threading"]
+==== Hyper-threading aware CPU manager policy (Technology Preview)
+Hyper-threading aware CPU manager policy in Openshift Container Platform is now available by default without the need for extra tuning. The cluster administrator can enable this feature if required. Hyper-threads are abstracted by the hardware as logical processors. Hyper-threading allows a single physical processor to execute two heavyweight threads (processes) at the same time, dynamically sharing processor resources.
```

Comment:
* s/hyper-threading/hyperthreading
* Hyphen in "Hyperthreading-aware"
* You say the feature is "available by default." But also that the "administrator can enable this feature if required." Can both be true?
* The word "hyperthread" doesn't quite sit right with me. Is that the right term? It doesn't seem to be used much, even in the Intel docs. But I don't know much at all about this. Just pointing it out to consider.

Also: Does this TP feature belong in the Technology Features tracker table?

Reply:
Yes, there was an entry in the TP table, now renamed to "Hyperthreading-aware CPU manager policy".

Reply:
Bullet points 1 and 2 are addressed. 
On bullet point 4, I will revisit the term if we integrate the upstream docs in a future release. 
Google seems equally divided on the issue of the hyphen between hyper and thread. This should even out one way or another as the technology is adopted.